client-app/shared/layout/components/login-form-section/login-form-section.vue

```diff
@@ -33,8 +33,15 @@
 </template>
 
 <script setup lang="ts">
+import { computed, inject } from "vue";
+import { configInjectionKey } from "@/core/injection-keys";
 import { SignInForm, useUser } from "@/shared/account";
+import { getImageUrl } from "@/ui-kit/utilities";
+
+const cfg = inject(configInjectionKey);
```

Comment:
Use `useThemeContext`. All of theme context was already injected at app init. So you can remove `computed` afterwords and use app settings as it is.